holmes/plugins/toolsets/azure_sql/tools/analyze_database_storage.py

```
@@ -0,0 +1,377 @@
+                    required=False,
+                ),
+                "top_tables": ToolParameter(
+                    description="Number of largest tables to analyze for space usage. Use 20 for comprehensive view, 10 for quick overview (default: 20)",
+                    type="integer",
+                    required=False,
+                ),
+            },
+            toolset=toolset,
+        )
+
+    def _build_storage_report(
```

Pylint (3.3.7)

[refactor] Too many local variables (48/15)

(R0914)